tgui/packages/tgui/interfaces/BandaMarines/MarineJobs.tsx

```diff
@@ -0,0 +1,372 @@
+  'Colonial Marshal Bureau': 'Colonial Marshal Bureau',
+  'Fax Responders': 'Секретари на Факсе',
+  Predators: 'Хищники',
+  Escaped: 'Сбежавшие',
+  Vehicles: 'Техника',
+  Animals: 'Животные',
+  Dead: 'Мёртвые',
+  Ghosts: 'Призраки',
+  Misc: 'Прочие',
+  NPCs: 'НИПы',
+  // MARK: Roles
+  Rifleman: 'Стрелок',
+  'Squad Leader': 'Командир Отряда',
+  'Combat Technician': 'Полевой Техник',
+  'Hospital Corpsman': 'Полевой Санитар',
+  'Weapons Specialist': 'Оружейный Специалист',
+  'Fireteam Leader': 'Командир Боевой Группы',
+  Smartgunner: 'Смартганнер',
+  // Survivors
+  Colonist: 'Колонист',
+  Passenger: 'Пассажир',
+  Survivor: 'Выживший',
+  'Synth Survivor': 'Выживший Синтетик',
+  'CO Survivor': 'Выживший КО',
+  'Any Survivor': 'Любой Выживший',
+  'Civilian Survivor': 'Выживший Гражданский',
+  'Security Survivor': 'Выживший Безопасник',
+  'Scientist Survivor': 'Выживший Ученый',
+  'Medical Survivor': 'Выживший Медик',
+  'Engineering Survivor': 'Выживший Инженер',
+  'Corporate Survivor': 'Выживший Корпорат',
+  'Hostile Survivor': 'Враждебный Выживший',
+
+  // Medical roles
+  'Chief Medical Officer': 'Главрач',
+  Doctor: 'Доктор',
+  Surgeon: 'Хирург',
+  'Field Doctor': 'Полевой Врач',
+  Nurse: 'Медработник',
+  Researcher: 'Исследователь',
+
+  // Civil roles
+  'Corporate Liaison': 'Связной Корпорации',
+  'Combat Correspondent': 'Полевой Корреспондент',
+  'Civilian Correspondent': 'Гражданский Корреспондент',
+  'Military Correspondent': 'Военный Корреспондент',
+
+  // Synthetic roles
+  Synthetic: 'Синтетик',
+  'Synthetic K9': 'Синтетик K9',
+  'Working Joe': 'Рабочий Джо',
+  'Hazmat Joe': 'Хазмат Джо',
+
+  // Command roles
+  'Commanding Officer': 'Коммандующий Офицер',
+  'Executive Officer': 'Исполнительный Офицер',
+  'Staff Officer': 'Штаб-Офицер',
+  'Auxiliary Support Officer': 'Офицер Поддержки',
+
+  // Dropship roles
+  'Gunship Pilot': 'Боевой Пилот',
+  'Dropship Pilot': 'Десантный Пилот',
+  'Tank Crew': 'Экипаж Танка',
+  'Dropship Crew Chief': 'Экипаж Десантного Корабля',
+  'Intelligence Officer': 'Офицер Разведки',
+
+  // Police roles
+  'Military Police': 'Военная Полиция',
+  'Military Warden': 'Военный Смотритель',
+  'Chief MP': 'Шеф Военной Полиции',
+
+  // Engineering roles
+  'Chief Engineer': 'Старший Инженер',
+  'Maintenance Technician': 'Техник Обслуживания',
+  'Ordnance Technician': 'Оружейный Техник',
+
+  // Requisition roles
+  Quartermaster: 'Квартирмейстер',
+  'Cargo Technician': 'Грузовой Техник',
+
+  // Marine Raider roles
+  'Marine Raider': 'Морпех Рейдер',
+  'Marine Raider Team Lead': 'Морпех Рейдер Командир',
+  'Marine Raider Platoon Lead': 'Морпех Рейдер Командир Взвода',
+
+  // Other roles
+  Stowaway: 'Безбилетник',
+  'USCM Marine': 'ККМП Морпех',
+  'USCM Colonel': 'ККМП Полковник',
+  'USCM Observer': 'ККМП Наблюдатель',
+  'USCM General': 'ККМП Генерал',
+  'Assistant Commandant of the Marine Corps':
+    'Помощник коменданта Корпуса Морской Пехоты',
```

Comment:
```suggestion
  'Assistant Commandant of the Marine Corps': 'Помощник коменданта Корпуса Морской Пехоты',
```
2 строки? Ну думаю ни на что не влияет